tests/e2e/cucumber.hooks.js

```diff
@@ -0,0 +1,31 @@
+const { Before, BeforeAll, AfterAll, After, setDefaultTimeout } = require('@cucumber/cucumber')
+const { firefox } = require('playwright')
+const { deleteFiles } = require('./utils/helpers')
+// default timeout (in milliseconds)
+setDefaultTimeout(100000)
```

Comment:
60 second timeout should be enough or was it not?
```suggestion
setDefaultTimeout(60 * 1000)
```